src/components/contextual/pages/pools/StakedPoolsTable.vue

```diff
@@ -12,14 +12,27 @@ import ProceedToSyncModal from '@/components/contextual/pages/vebal/cross-chain-
 import { providePoolStaking } from '@/providers/local/pool-staking.provider';
 
 import PortfolioSyncTip from '../vebal/cross-chain-boost/PortfolioSyncTip.vue';
+import { useCrossChainSync } from '@/providers/cross-chain-sync.provider';
+
+import CheckpointGaugeModal from '../vebal/cross-chain-boost/CheckpointGaugeModal.vue';
+import CheckpointAllGaugesModal from '../vebal/cross-chain-boost/CheckpointAllGaugesModal.vue';
 
 /**
  * STATE
  */
 const showUnstakeModal = ref(false);
 const poolToUnstake = ref<Pool | undefined>();
+
+const showRestakeModal = ref(false);
+const poolToRestake = ref<Pool | undefined>();
+
 const showProceedModal = ref(false);
+const defaultPoolActions = ['unstake', 'add', 'vote'];
```

Comment:
Can you define this in one place, please? I see it's also defined in `UnstakedPoolsTable.vue`

Reply:
default actions are different for staked and unstaked pools